src/footer/_footer.scss

```diff
@@ -0,0 +1,172 @@
+    @include govuk-font-regular;
+    @include govuk-typography-responsive($govuk-font-16);
+    @include govuk-responsive-padding($govuk-spacing-responsive-7, "top");
+    @include govuk-responsive-padding($govuk-spacing-responsive-5, "bottom");
+
+    border-top: 1px solid $govuk-footer-border-top;
+    color: $govuk-footer-text;
+    background: $govuk-footer-background;
+  }
+
+  .govuk-c-footer__link {
+    @include govuk-focusable-fill;
+
+    &:link,
+    &:visited {
+      color: $govuk-footer-link;
+    }
+
+    &:hover,
+    &:active {
+      color: $govuk-footer-link-hover;
+    }
+  }
+
+  .govuk-c-footer__section-break {
+    margin: 0; // Reset `<hr>` default margins
+    @include govuk-responsive-margin($govuk-spacing-responsive-8, "bottom");
+    border: 0; // Reset `<hr>` default borders
+    border-bottom: 1px solid $govuk-footer-border;
+  }
+
+  .govuk-c-footer__meta {
+    display: flex; // Support: Flexbox
+    margin-right: -$govuk-gutter-half;
+    margin-left: -$govuk-gutter-half;
+    flex-wrap: wrap; // Support: Flexbox
+    align-items: flex-end; // Support: Flexbox
+    justify-content: center; // Support: Flexbox
+  }
+
+  .govuk-c-footer__meta-item {
+    margin-right: $govuk-gutter-half;
+    margin-bottom: $govuk-spacing-scale-5;
+    margin-left: $govuk-gutter-half;
+  }
+
+  .govuk-c-footer__meta-item--grow {
+    flex: 1; // Support: Flexbox
+    @include mq ($until: tablet) {
+      flex-basis: 320px; // Support: Flexbox
+    }
+  }
+
+  .govuk-c-footer__licence-logo {
+    display: inline-block;
+    margin-right: $govuk-spacing-scale-2;
+    @include mq ($until: desktop) {
+      margin-bottom: $govuk-spacing-scale-3;
+    }
+    vertical-align: top;
+  }
+
+  .govuk-c-footer__licence-description {
+    display: inline-block;
+  }
+
+  .govuk-c-footer__copyright-logo {
+    display: inline-block;
+    min-width: $govuk-footer-crest-image-width;
+    padding-top: ($govuk-footer-crest-image-height + $govuk-spacing-scale-2);
+    background-image: govuk-file-url("govuk-crest.png");
+    @include govuk-h-device-pixel-ratio {
+      background-image: govuk-file-url("govuk-crest-2x.png");
+    }
+    background-repeat: no-repeat;
+    background-position: 50% 0%;
+    background-size: $govuk-footer-crest-image-width $govuk-footer-crest-image-height;
+    text-align: center;
+    text-decoration: none;
+    white-space: nowrap;
+  }
+
+  .govuk-c-footer__inline-list {
+    margin-top: 0;
+    margin-bottom: $govuk-spacing-scale-3;
+    padding: 0;
+  }
+
+  .govuk-c-footer__inline-list-item {
+    display: inline-block;
+    margin-right: $govuk-spacing-scale-3;
+    margin-bottom: $govuk-spacing-scale-1;
+  }
+
+  .govuk-c-footer__heading {
+    @include govuk-responsive-margin($govuk-spacing-responsive-7, "bottom");
+    padding-bottom: $govuk-spacing-scale-4;
+    @include mq ($until: tablet) {
+      padding-bottom: $govuk-spacing-scale-2;
+    }
+    border-bottom: 1px solid $govuk-footer-border;
```

Comment:
This heading bottom border doesn't currently exist on GOV.UK footer on mobile (see screen grab). This might have been a design decision though? I think it's possibly an improvement on the existing thing.

![screen shot 2018-03-07 at 14 50 21](https://user-images.githubusercontent.com/5007934/37098863-e156b6da-2216-11e8-96b5-47132ec08104.png)

Reply:
I figured that was a mistake so wanted to make the new version more consistent.

Reply:
That's cool. It does look better now compared to the current slightly bunched up mobile view 👍

Reply:
Thank @dashouse spacing scale 🌈